pkg/cli/admin/prune/imageprune/prune.go

```diff
@@ -276,7 +266,7 @@ type Pruner interface {
 // pruner is an object that knows how to prune a data set
 type pruner struct {
 	usedTags   map[imageStreamTagReference][]resourceReference
-	usedImages map[imageStreamImageReference][]resourceReference
+	usedImages map[string][]resourceReference
```

Comment:
It tracks ImageStreamImages. It should protect historical records in imagestreams.